WooCommerce/WooCommerceTests/Model/AddressWooTests.swift

```diff
@@ -14,7 +14,7 @@ class AddressWooTests: XCTestCase {
 
     func test_from_taxRate_when_tax_rate_has_location_then_prioritizes_array_properties() {
         // Given
-        let taxRate = TaxRate.fake().copy(country: "US", state: "CA", postcode: "12345", postcodes: ["9999"], city: "Miami", cities: ["New York"])
+        let taxRate = TaxRate.fake().copy(state: "CA", country: "US", postcode: "12345", postcodes: ["9999"], city: "Miami", cities: ["New York"])
```

Comment:
[these `TaxRate` changes](https://github.com/woocommerce/woocommerce-ios/pull/10652/files#r1319410614) shouldn't be needed on the latest `trunk`